packages/core/src/split-layout/SplitLayout.tsx

```diff
@@ -0,0 +1,70 @@
+import { ElementType, forwardRef, ReactElement, ReactNode } from "react";
+import { FlexLayout, FlexLayoutProps } from "../flex-layout";
+import {
+  PolymorphicComponentPropWithRef,
+  PolymorphicRef,
+  useIsViewportLargerThanBreakpoint,
+} from "../utils";
+import { Breakpoints } from "../breakpoints";
+
+export type SplitLayoutProps<T extends ElementType> =
+  PolymorphicComponentPropWithRef<
+    T,
+    {
+      /**
+       * Defines the default behavior for how flex items are laid out along the cross axis on the current line.
+       */
+      align?: FlexLayoutProps<ElementType>["align"];
+      /**
+       * Establishes the main-axis, defining the direction children are placed. Default is "row".
+       */
+      direction?: FlexLayoutProps<ElementType>["direction"];
+      /**
+       * End component to be rendered.
+       */
+      endItem?: ReactNode;
+      /**
+       * Controls the space between left and right items.
+       */
+      gap?: FlexLayoutProps<ElementType>["gap"];
+      /**
+       * Start component to be rendered.
+       */
+      startItem?: ReactNode;
+      /**
+       * Breakpoint at which the horizontal split wraps.
+       */
+      wrapAtBreakpoint?: keyof Breakpoints;
```

Comment:
We should make sure this prop name is consistent across components with similar behaviour.

I think for the user this might not feel like 'wrapping' even though that's what the Flex is doing under the hood. What do you think?

Reply:
I agree, not very sure about this name, it's not even wrapping, it is changing direction 🤔

Reply:
See https://github.com/jpmorganchase/salt-ds/pull/1035#discussion_r1097408918. We could remove both wrap and wrapAtBreakpoint?